Formula/tg.rb

```diff
@@ -0,0 +1,28 @@
+class Tg < Formula
+  include Language::Python::Virtualenv
+
+  desc "Terminal telegram client"
+  homepage "https://github.com/paul-nameless/tg"
+  url "https://github.com/paul-nameless/tg/archive/refs/tags/v0.11.0.tar.gz"
+  sha256 "2303983af7b7246fb3198bcae688d3b6327a96bbfc91c94ef9bc783cc7191f42"
+  license "Unlicense"
+
+  bottle :unneeded
```

Comment:
Yes it is

Reply:
Is it needed to change? I didn't understand the action that comment suggest.

Reply:
You're saying no bottle is needed, but this wil need a bottle. You should remove this.
```suggestion
```